pom.xml

```diff
@@ -37,6 +37,13 @@
     </dependencyManagement>
 
     <dependencies>
+
+        <dependency>
+            <groupId>com.querydsl</groupId>
+            <artifactId>querydsl-apt</artifactId>
+            <version>${querydsl}</version>
```

Comment:
Should be `<scope>provided</scope>` to avoid mandatory dependencies on the annotation processor.